packages/core/components/QuerySidebar/tutorials/index.ts

```diff
@@ -23,6 +23,26 @@ export const HELP_OPTIONS = (dispatch: Dispatch): IContextualMenuItem[] => {
             href: "https://alleninstitute.github.io/aics-fms-file-explorer-app/",
             target: "_blank",
         },
+        {
+            key: "gh-issues-page",
+            text: "Submit Issue",
+            iconProps: {
+                iconName: "Bug",
+            },
+            title: "Opens the FMS File Explorer GH issues page",
```

Comment:
I lean towards spelling out GitHub for non SWEs but that's just a personal preference.
I'm also considering the wording of the menu text. Would "Report an Issue" or something similar be more universally familiar than submit?

Reply:
updated in [b413566](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/97/commits/b413566106d632894c81a1d9c5bbf783a52f000e)